apps/vaporgui/PMultiVarSelector.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "PWidget.h"
+
+class QListWidget;
+class QListWidgetItem;
+
+//! \class PMultiVarSelector
```

Comment:
Please add documentation wrt how the class decides what variables to present.

Reply:
Done